core/lib/vm_executor/src/oneshot/contracts.rs

```diff
@@ -107,9 +107,9 @@ impl<C: ContractsKind> MultiVmBaseSystemContracts<C> {
             ProtocolVersionId::Version25 => &self.vm_protocol_defense,
             ProtocolVersionId::Version26 => &self.gateway,
             ProtocolVersionId::Version27 => &self.vm_evm_emulator,
-
-            // Speculative base system contracts for the next protocol version to be used in the upgrade integration test etc.
             ProtocolVersionId::Version28 => &self.vm_evm_emulator,
```

Comment:
I didn't mean to just remove TODO, you should add add new multivm bootloaders and use `vm_ec_precompiles` here and for v29